docs/sources/operators-guide/architecture/components/ruler/index.md

```diff
@@ -14,18 +14,35 @@ Each tenant has a set of recording and alerting rules and can group those rules
 
 ![Architecture of Grafana Mimir's ruler component](ruler.svg)
```

Comment:
This diagram applies only to the internal operational mode. I would move under "Internal".